### PR TITLE
[Unity][Op] add einsum and flip in Relax

### PR DESCRIPTION
**Pull request by yongwww**
Add einsum and flip in Relax's Op and Legalizer, we met them in model like tiny GPT-J

cc: @sunggg @jwfromm @slyubomirsky @psrivas2 @tqchen

### CODE REVIEW

**Review by sunggg — APPROVED** [review]
LGTM! Thank you!

**Review by tqchen — APPROVED** [review]

File: python/tvm/relax/op/manipulate.py [hunk 1]
```diff
@@ -441,6 +441,56 @@ def cumsum(data: Expr, axis: Optional[int] = None, dtype: Optional[Union[str, Da
     return _ffi_api.cumsum(data, axis, dtype)  # type: ignore
 
 
+def einsum(operands, subscripts):
```
Comment (tqchen) [inline python/tvm/relax/op/manipulate.py:444]: einsum should be part of linear algebra

File: include/tvm/relax/attrs/manipulate.h [hunk 2]
```diff
@@ -140,6 +140,25 @@ struct CumsumAttrs : public tvm::AttrsNode<CumsumAttrs> {
   }
 };  // struct CumsumAttrs
```
Comment (tqchen) [inline include/tvm/relax/attrs/manipulate.h:141]: cumsum should be moved to statistical

**Review by tqchen — CHANGES_REQUESTED** [review]

**Comment by tqchen** [comment]
Thanks @yongwww some organizational comments

**Review by tqchen — APPROVED** [review]

**Pull request merged**